MSAL/src/public/configuration/MSALPublicClientApplicationConfig.h

```diff
@@ -110,7 +116,22 @@ to target MSAL at a specific test slice & flight. These apply to all requests ma
  */
 - (nonnull instancetype)initWithClientId:(NSString *)clientId
                              redirectUri:(nullable NSString *)redirectUri
-                               authority:(nullable MSALAuthority *)authority NS_DESIGNATED_INITIALIZER;
+                               authority:(nullable MSALAuthority *)authority;
+
+/**
+ Initialize a MSALPublicClientApplicationConfig with a given clientId and a nested clientid
+ 
+ @param  clientId       The clientID of your application, you should get this from the app portal.
+ @param  redirectUri    The redirect URI of the application
+ @param  authority      The target authority
+ @param  nestedAuthBrokerClientId     The clientID of your child application
+ @param  nestedAuthBrokerRedirectUri    The redirect URI of the child application
+ */
+- (nonnull instancetype)initWithClientId:(NSString *)clientId
+                             redirectUri:(nullable NSString *)redirectUri
+                               authority:(nullable MSALAuthority *)authority
+                nestedAuthBrokerClientId:(nullable NSString *)nestedAuthBrokerClientId
+             nestedAuthBrokerRedirectUri:(nullable NSString *)nestedAuthBrokerRedirectUri NS_DESIGNATED_INITIALIZER;
```

Comment:
I don't think this should be a designated initializer. I don't expect any 3P app to use this API

Reply:
Updated